### ISSUE TITLE
Performance issue due to tooltips

### ISSUE THREAD

**Issue by mickaelchanrion** [vidstack/player#1386]
Hi there 👋
Thanks for this great player!

### Current Behavior:

On a website I am working on, I have pages with around 10 vidstack instances and I noticed a drop of frames while scrolling. I'm using 1.12.5

After recording the performance tab, I could see that functions are triggered on scroll event. Functions coming from floating-ui, a dependency of vidstack. More specifically, the `autoUpdate` function in use [here](https://github.com/vidstack/player/blob/next/packages/vidstack/src/utils/dom.ts#L121).

Floating UI's docs [recommends](https://floating-ui.com/docs/autoupdate#usage) to call the cleanup function returned by `autoUpdate` whenever the handled element is removed (understand "is not visible"?) to avoid severe performance degradation.

In vidstack, it seems like this cleanup function is called by the [`onDispose` hook](https://github.com/vidstack/player/blob/next/packages/vidstack/src/components/ui/tooltip/tooltip-content.ts#L49). I'm not familiar with `maverick.js` but my guess is this is only called when the tooltip is removed from the dom and this would explain the performance drop on scroll.

<img width="1512" alt="image (3)" src="https://github.com/user-attachments/assets/37dd4e40-cf82-409b-8e0d-73ffa95ca245">

### Expected Behavior:

Could it be possible to change how things are handled?
To sum up:
- call autoUpdate on show
- call cleanup on hide

If so, both tooltip-content and menu-item components are concerned by this matter.

### Steps To Reproduce:

1. Instantiate a vidstack player
2. Record performance tab
3.  Scroll
4. See that floating-ui functions (detectOverflow, getClippingRect, etc.) are called even when tooltips are not visible

Thank you so much for reading me. :)

**Comment by mihar-22** [comment]
I can't believe I missed that, thank you so much for catching it and the detailed report! Fixed and the release will be rolling out momentarily.

**Comment by mickaelchanrion** [comment]
Just checked. It's so much better now!
Thank you so much for fixing the issue this fast! 🚀
